transform/mattermost-analytics/models/intermediate/hightouch/int_stripe_onprem_subscriptions.sql

```diff
@@ -0,0 +1,33 @@
+{{config({
+    "materialized": 'table',
+    "unique_key":"invoice_id",
+  })
+}}
+
+WITH payment AS (
+    SELECT
+        invoices.charge,
+        invoices.subscription,
+        'ONL' || invoices.number AS invoice_number,
+        invoices.number as stripe_invoice_number,
+        invoices.id as invoice_id,
+        invoices.created_at as invoice_created_at,
+        ROW_NUMBER() OVER (PARTITION BY invoices.subscription ORDER BY invoices.created_at) as row_num
```

Comment:
This column is not used after joining. It can be removed.

Reply:
This is now dropped in the new modification of the code.